commands.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+def create(schima):
```

Comment:
What would happen if the directory was already created, will the program crash or should it alert the user without crashing?